src/mlir/dialect/VeronaDialect.td

```diff
@@ -27,18 +27,9 @@ class Verona_Op<string mnemonic, list<OpTrait> traits = []> :
     let verifier = [{ return ::verify(*this); }];
 }
 
+class Verona_TypeDef<string name> : TypeDef<Verona_Dialect, name> { }
```

Comment:
Does this mean "any type defined in the verona dialect is a verona type"?

Reply:
Sorry I don’t really understand the question. The switch to ODS does not affect the implementation of `isaVeronaType`, which is roughly how we define what a “Verona Type”.

To be honest I don’t really know what impact the dialect we pass as the first argument of `TypeDef` has. I’ll try and dig into that.

Reply:
I was under the impression that the type attribute below (now removed) had some implicit connection with the type storage defined for each of our internal "Verona types", and that somehow the new typedef was replacing it.

Maybe they're two independent changes?